src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/config/IntegrationTest.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config
+
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+
+/**
+ * ## Integration test annotation.
+ * This annotation is meant for all the integration tests.
+ * Its purpose is to:
+ *  - centralise the integration test configuration
+ *  - avoid copy and past
+ *  - enable one profile for efficient running only @DirtysContext will restart/refresh the orchestrator context.
+ */
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureWireMock
+@ActiveProfiles("integration")
+@ContextConfiguration(initializers = [IntegrationTestConfig::class])
+internal annotation class IntegrationTest
```

Comment:
Isn't this on master already?

Reply:
hmmm ... this was from a branch where I moved the untility classes to a package to clean up the integration tests.
I put the wrong branch into PR simplify_integration_test was supposed to be now its a little bit messed up ... give me a sec an Ill try unwind this.

Reply:
DONE :smile: